sympy/categories/diagram_drawing.py

```diff
@@ -2089,7 +2089,7 @@ def count_morphisms_filtered(dom, cod, curving):
         # Let's now get the name of the morphism.
         morphism_name = ""
         if isinstance(morphism, IdentityMorphism):
-            morphism_name = "id_{%s}" + latex(obj)
+            morphism_name = "id_{%s}" + latex(grid[i,j])
```

Comment:
I do not think this code is ever executed. It looks like IdentityMorphism:s are removed from the morphisms provided to the method. At least that is what I believe after trying and also reading a bit of the documents. Still, not being an expert it seems to make sense to keep it(?).

grid[i, j] will at least provide the name of the object if ever called. obj will lead to an error.